likecoin/admin/matters.php

```diff
@@ -166,11 +166,13 @@ function likecoin_handle_matters_api_error( $error ) {
  * @param WP_Post| $post Post object.
  */
 function likecoin_get_post_tags_for_matters( $post ) {
-	$post_id   = $post->ID;
-	$func      = function( $terms ) {
-		return $terms->name;
+	$post_id              = $post->ID;
+	$raw_encode_post_tags = array();
```

Comment:
Please remove `$raw_encode_post_tags` if it is not used

Reply:
ok!